plugins/module_utils/_global_catalog.py

```diff
@@ -156,3 +217,5 @@ def _get_idcams_cmd_gcd(dataset):   # type: (dict) -> dict
 CI_PERCENT = 10
 CA_PERCENT = 10
 SHARE_CROSSREGION = 2
+MAX_LINE_LENGTH = 72
```

Comment:
I know the existing constants were here, I'm wondering if we should separate these out a bit as these 2 new ones are not part of the IDCAMS statement unlike all the other. A simple newline and comment might suffice, but also feel free to rearrange them in the file, whichever makes sense